app/models/bookmark.rb

```diff
@@ -6,6 +6,7 @@ class Bookmark < ApplicationRecord
 
   belongs_to :bookmarkable, polymorphic: true, inverse_of: :bookmarks
   belongs_to :pseud
```

Comment:
There are [a few different ways of adding presence validations](https://discuss.rubyonrails.org/t/is-it-ever-ok-to-validate-the-presence-of-a-primary-key-of-a-belongs-to-association/84904), but let's use:
```rb
belongs_to :pseud, optional: false
```
and let Rails [optimize when the validation should be run](https://github.com/rails/rails/blob/d39db5d1891f7509cde2efc425c9d69bbb77e670/activerecord/lib/active_record/associations/builder/belongs_to.rb#L129-L138) (once we get to Rails 7.1).